.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Part of the project requirement is to list the `node_modules` and the `test.md` file in the `.gitignore` file. All you need do is to include `test.md` in this file. This is to ensure you practice and master how to intentionally specify untracked files that Git should ignore.